kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -31,8 +31,8 @@ class ConnectWrapper:
     Wraps Kafka Connect APIs
     """
 
-    def __init__(self, config: KpopsConfig) -> None:
-        self._url: AnyHttpUrl = config.kafka_connect.url
+    def __init__(self, kafka_connect_config: KafkaConnectConfig) -> None:
+        self._url: AnyHttpUrl = kafka_connect_config.url
```

Comment:
keep the `config` name since it's already specific to this class and assign it as instance attribute instead of only `url`